src/_guides/libraries/index.md

```diff
@@ -59,33 +59,52 @@ The following table lists the Dart core libraries that work on the
 |-----------------------------------------------+-------------------------------|
 | Library                                       | Notes   |
 |-----------------------------------------------|-------------------------------|
-| [`dart:html`][dart-html]                <br> HTML elements and other resources for web-based applications. | |
-| [`dart:indexed_db`][dart-indexed_db]    <br> Client-side key-value store with support for indexes. | |
-| [`dart:js`][dart-js]                    <br> _Don't use._ Instead, use the `js` package, as described in [JavaScript interoperability][]. | *DEPRECATED* |
-| [`dart:js_util`][dart-js_util]          <br> APIs to supplement missing functionality in `dart:html` or the `js` package. | |
-| [`dart:svg`][dart-svg]                  <br> Scalable Vector Graphics. | |
-| [`dart:web_audio`][dart-web_audio]      <br> High-fidelity audio programming in the browser. | |
-| [`dart:web_gl`][dart-web_gl]            <br> 3D programming in the browser. | |
+| [`dart:html`][dart-html], [`package:html`][package-html]<br>HTML elements and other resources for web-based applications.<br>`package:html` provides an HTML parser. | |
+| [`dart:indexed_db`][dart-indexed_db]<br>Client-side key-value store with support for indexes. | |
+| [`dart:js`][dart-js], [`package:js`][package-js]<br>_Don't use `dart:js` directly._ Instead, use the `package:js`, as described in [JavaScript interoperability][]. | Direct use of `dart:js` is *deprecated* |
```

Comment:
Is it really *deprecated* (we intend to remove the possibility) or is it just *discouraged and unsupported*?